micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -125,30 +117,110 @@
+          });
+        }
+        return messageMap;
+      },
+    },
+  };
+
+  const { data: localizationMap, isLoading: isLocalizationLoading } = Digit.Hooks.useCustomAPIHook(localizationReqCriteria);
+
+  // Transform data if localizationMap is available
+  let finalData = data;
+  if (data && data.data && localizationMap && localisableFields.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)